src/components/BreadcrumbTrail.tsx

```diff
@@ -56,7 +58,12 @@ const BreadcrumbTrail = () : JSX.Element | null => {
 
     return (
         <div className={ classes.root }>
-            <Breadcrumbs aria-label="breadcrumb" maxItems={ isSmallScreen ? 2 : 6 } separator={ <NavigateNextIcon fontSize="small" /> }>
+            <Breadcrumbs
+                aria-label="breadcrumb"
+                itemsAfterCollapse={ smallScreen ? 1 : mediumScreen ? 2 : 3 }
+                itemsBeforeCollapse={ smallScreen ? 1 : mediumScreen ? 2 : 3 }
+                maxItems={ smallScreen ? 2 : mediumScreen ? 4 : largeScreen ? 6 : 10 }
```

Comment:
I'm not loving the nested ternaries but will accept it because both the conditions and the expressions are very simple.